tools/oesign/oesign.c

```diff
@@ -738,19 +778,24 @@ int oedigest(const char* enclave, const char* conffile, const char* digest_file)
     oe_sgx_enclave_properties_t properties;
     OE_SHA256 mrenclave = {0};
     OE_SHA256 digest = {0};
+    uint32_t miscselect = 0;
```

Comment:
Original code initialize miscselect to SGX_SIGSTRUCT_MISCSELECT which is 0. If we not using SGX_SIGSTRUCT_MISCSELECT, please remove the definition.

Reply:
The implementation now does not modify the SIGSTRUCT.MISCSELECT data, only the SECS.MISCSELECT, so I reserve the definition for now.